syntax = "proto3";

package max.dillon;

message TreeSearchResult {
    uint32 index = 1;
    float prob = 2;
}

message TrainingInstance {
    bytes board_state = 1;
    bool white_move = 2;
    uint32 outcome = 3;
    uint32 game_length = 4;
    repeated TreeSearchResult tree_search_result = 5;
}

